lib/plugin.js

```diff
@@ -2,7 +2,9 @@ import Vue from 'vue'
 import VueAnalytics from 'vue-analytics'
 
 export default async (ctx, inject) => {
-  const options = <%= serialize(options) %>
+  const runtimeConfig = ctx.$config && ctx.$config.analytics || {}
```

Comment:
Top-level key is `googleAnalytics` would be better using it for consistency :)

Reply:
Good point